krun/platform.py

```diff
@@ -221,7 +221,8 @@ def bench_cmdline_adjust(self, args, env_dct):
         changes = self.bench_env_changes()
         EnvChange.apply_all(changes, combine_env)
 
-        return self.adjust_env_cmd(combine_env) + args
+        return self.adjust_env_cmd(combine_env) + \
```

Comment:
The docs for this method (which GitHub won't let me comment on directly) still say `libkruntime` rather than `libkrun`.